test/runtests.jl

```diff
@@ -39,6 +40,13 @@ end
     @test free(y) === nothing
 
     @test_throws ArgumentError malloc(Vector{Int}, 10)
+    
+    z = malloc(Int, 4, 6, 10)
+    @test length(z) == 240
+    @test size(z) == (4, 6, 10)
+    @test strides(z) == (1, 4, 24)
+    @test z isa PtrArray
+    @test z isa DenseArray{Int, 3}
```

Comment:
It's good practice to free :)
```suggestion
    @test z isa DenseArray{Int, 3}
    free(z)
```

Reply:
hahahaha my hide-and-seek-game needs some work :)